src/ExceptionAll.Client/Services/ValidationService.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ExceptionAll.Client.Services;
+
+public class ValidationService : IValidationService
+{
+    private readonly JsonSerializerOptions _options = new(JsonSerializerDefaults.Web);
+
+    /// <exception cref="ExceptionAllException"/>
+    public async ValueTask ValidateAsync(
+        HttpResponseMessage message)
+    {
+        var content = await message.Content.ReadAsStringAsync();
+
+        try
+        {
+            var details = JsonSerializer.Deserialize<ApiErrorDetails>(content, _options);
```

Comment:
You are not using your serializer service here

Reply:
Due to removal of validation service, this one should be invalid